packages/field-base/test/field-mixin.test.js

```diff
@@ -369,20 +400,21 @@ describe('field-mixin', () => {
         expect(element.hasAttribute('has-helper')).to.be.true;
       });
 
-      it('should not update slotted helper content on property change', () => {
+      it('should not update slotted helper content on property change', async () => {
         element.helperText = '3 digits';
+        await nextFrame();
         expect(helper.textContent).to.not.equal('3 digits');
       });
     });
 
     describe('slotted with property', () => {
       beforeEach(async () => {
         element = fixtureSync(`
-          <field-mixin-element helper-text="Default">
+          <${tag} helper-text="Default">
             <div slot="helper">Custom</div>
-          </field-mixin-element>
+          </${tag}>
         `);
-        await nextFrame();
+        await nextRender();
```

Comment:
**question:** Why did it require using `nextRender` instead of `nextFrame` here?